### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by sjiveson**
Major improvements to the readme to provide more detail and clarity on the additional steps required to get going.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/piwik/docker-piwik.svg?branch=master)](https://travis-ci.org/piwik/docker-piwik)
 
-Piwik is the leading open-source analytics platform that gives you more than just powerful analytics:
+Piwik is the leading open-source analytics platform that gives you much more than powerful visitor metrics:
```
Comment (pierreozoux) [inline README.md]: Arf, it is form the piwik.org :)

File: README.md [hunk 2]
```diff
@@ -12,22 +12,35 @@ Piwik is the leading open-source analytics platform that gives you more than jus
 
 ![logo](https://rawgit.com/piwik/docker-piwik/master/logo.svg)
 
-# How to use this image
+## Usage
+
+In keeping with a 'pure' micro-services approach, this image runs a Piwik service only (in the form of FastCGI). Because of that it **must** be used with companion containers which provide a database for data storage and HTTP to FastCGI proxy/translation services for the user interface.
+
+## Pulling
```
Comment (pierreozoux) [inline README.md]: pulling is unnecessary step as the run will pull if not present.

File: README.md [hunk 3]
```diff
@@ -12,22 +12,35 @@ Piwik is the leading open-source analytics platform that gives you more than jus
...
+sudo docker pull piwik:latest
+```
+
+## Runtime
+
+You can run the Piwik container and service like so:
 
 ```bash
-docker run --link some-mysql:db -d piwik
+docker run -d --link some-mysql:db piwik:latest
```
Comment (pierreozoux) [inline README.md]: actually, looking at other official images, we can remove the latest and the -d all together for clarity purpose.

**Comment by pierreozoux** [comment]
Great, indeed, it sounds a lot better \o/

Could you address the comments and squash please, then I'd be more than happy to merge :)

**Comment by pierreozoux** [comment]
Thanks!

If you squash, you'd make my life a lot easier :)

https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

If you can't, I'll do it myself :)

**Comment by sjiveson** [comment]
Finally got there.

**Comment by sjiveson** [comment]
By the way, I didn't remove the -d option as I think it's necessary. There's no point running the piwik container itself in interactive mode.

**Comment by pierreozoux** [comment]
Thanks a lot for your time :)

**Comment by sjiveson** [comment]
You're welcome, should hopefully save others plenty of their time and you answering questions.

**Pull request merged**